mobile/lib/shared/ui/immich_image.dart

```diff
@@ -49,7 +49,7 @@ class ImmichImage extends StatelessWidget {
     final Asset asset = this.asset!;
     if (useLocal(asset)) {
       return Image(
-        image: localThumbnailProvider(asset),
+        image: localThumbnailProvider(asset, dimension: 2048),
```

Comment:
This should probably fix it. I think I'll make it a point to create a new `localImageProvider` which doesn't need a dimension, and just uses the full asset. Because using the thumbnail provider to show the full image feels a bit dirty.